src/Symfony/Component/Form/AbstractType.php

```diff
@@ -52,7 +52,7 @@ public function configureOptions(OptionsResolver $resolver)
      */
     public function getBlockPrefix()
     {
-        return StringUtil::fqcnToBlockPrefix(\get_class($this));
+        return StringUtil::fqcnToBlockPrefix(\get_class($this)) ?: '';
```

Comment:
We know for sure that it will be either string or null.
```suggestion
        return StringUtil::fqcnToBlockPrefix(\get_class($this)) ?? '';
```

Reply:
I've got used to it too much and forgot that it's since 7 only :)